web/themes/custom/cloudy/package-lock.json

```diff
@@ -4372,8 +4372,7 @@
         "ansi-regex": {
           "version": "2.1.1",
           "bundled": true,
-          "dev": true,
```

Comment:
Must have been changed by either npm install or npm run dev

Reply:
It would be NPM install... I just remembered there was some history to this. Your NPM cache is out of date. Follow the steps in this issue: https://github.com/npm/npm/issues/20106#issuecomment-383770315.

You'll need to remove the package-lock.json from your local, npm cache clean, then npm install to fix the issue. Otherwise every commit you make will try and change the package-lock.json.